mmv1/products/tpuv2/Vm.yaml

```diff
@@ -88,3 +94,220 @@ properties:
     name: 'description'
     description: |
       Text description of the TPU.
+  - !ruby/object:Api::Type::String
+    name: 'cidrBlock'
+    default_from_api: true
+    description: |
+      The CIDR block that the TPU node will use when selecting an IP address. This CIDR block must
+      be a /29 block; the Compute Engine networks API forbids a smaller block, and using a larger
+      block would be wasteful (a node can only consume one IP address). Errors will occur if the
+      CIDR block has already been used for a currently existing TPU node, the CIDR block conflicts
+      with any subnetworks in the user's provided network, or the provided network is peered with
+      another network that is using that CIDR block.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'networkConfig'
+    default_from_api: true
+    immutable: true
+    description: |
+      Network configurations for the TPU node.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'network'
+        immutable: true
+        default_from_api: true
+        description: |
+          The name of the network for the TPU node. It must be a preexisting Google Compute Engine
+          network. If none is provided, "default" will be used.
+      - !ruby/object:Api::Type::String
+        name: 'subnetwork'
+        immutable: true
+        default_from_api: true
+        description: |
+          The name of the subnetwork for the TPU node. It must be a preexisting Google Compute
+          Engine subnetwork. If none is provided, "default" will be used.
+      - !ruby/object:Api::Type::Boolean
+        name: 'enableExternalIps'
+        default_from_api: true
+        immutable: true
+        send_empty_value: true
+        description: |
+          Indicates that external IP addresses would be associated with the TPU workers. If set to
+          false, the specified subnetwork or network should have Private Google Access enabled.
+      - !ruby/object:Api::Type::Boolean
+        name: 'canIpForward'
+        default_from_api: true
+        immutable: true
+        send_empty_value: true
+        description: |
+          Allows the TPU node to send and receive packets with non-matching destination or source
+          IPs. This is required if you plan to use the TPU workers to forward routes.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'serviceAccount'
```

Comment:
I think we can safely mark this immutable- it can still safely be removed from config because it's `default_from_api`, and any other change will recreate it.

Reply:
Will do!